nncf/config/config.py

```diff
@@ -94,17 +98,11 @@ def get_redefinable_global_param_value_for_algo(self, param_name: str, algo_name
     def validate(loaded_json):
         COMMON_REF_VS_ALGO_SCHEMA = {**REF_VS_ALGO_SCHEMA, **EXPERIMENTAL_REF_VS_ALGO_SCHEMA}
         ROOT_NNCF_CONFIG_SCHEMA = get_root_nncf_config_schema(COMMON_REF_VS_ALGO_SCHEMA)
-        try:
-            jsonschema.validate(loaded_json, schema=ROOT_NNCF_CONFIG_SCHEMA)
-        except jsonschema.ValidationError as e:
-            logger.error('Invalid NNCF config supplied!')
+        NNCFConfig._validate_json_section_by_schema(loaded_json, ROOT_NNCF_CONFIG_SCHEMA)
 
-            # The default exception's __str__ result will contain the entire schema,
-            # which is too large to be readable.
-            import nncf.config.schema as config_schema
-            msg = e.message + '. See documentation or {} for an NNCF configuration file JSON schema definition'.format(
-                config_schema.__file__)
-            raise jsonschema.ValidationError(msg)
+        bootstrap_nas_section = loaded_json.get(BOOTSTRAP_NAS_ALGO_NAME_IN_CONFIG)
+        if bootstrap_nas_section is not None:
+            NNCFConfig._validate_json_section_by_schema(bootstrap_nas_section, BOOTSTRAP_NAS_SCHEMA)
```

Comment:
Because BootstrapNAS is not a compression algorithm with builder and controller - it's built on top of elasticity and training compression algorithms + search part. It also assumes a different integration as opposed to ordinary compression algorithms. That's why it can be added to `EXPERIMENTAL_REF_VS_ALGO_SCHEMA`.

It can be a part of `ROOT_NNCF_CONFIG_SCHEMA` like AA algo:
```json
"accuracy_aware_training": with_attributes(ACCURACY_AWARE_TRAINING_SCHEMA, ...)
```
But this schema shouldn't include experimental attributes.

Reply:
Yes, it's possible. I've generalized it. Thanks!